index.d.ts

```diff
@@ -0,0 +1,32 @@
+type Mode = "binary" | "metric" | "decimal" | "compatibility";
+
+interface ParseOptions {
+    mode?: Mode;
+}
+
+declare function parse(value: number | string, options?: ParseOptions): number | null;
+
+interface FormatOptions {
+    decimalPlaces?: number;
+    fixedDecimals?: number;
+    mode?: Mode;
+    thousandsSeparator?: string;
+    unit?: string;
+    unitSeparator?: string;
+}
+
+declare function format(value: number, options?: FormatOptions): string | null;
+
+declare function withDefaultMode(mode: Mode): BytesIec;
+
+type BytesIec = typeof format & {
```

Comment:
Lets rename this to `BytesFormatter` instead since its the function/module mix

Reply:
But it also parses, which is opposite of a formatter.

Reply:
True True… I just don't like the noun form, how about:  `ByteConverter` then